airflow/providers/docker/hooks/docker.py

```diff
@@ -87,7 +87,7 @@ def get_conn(self) -> APIClient:
         self.__login(client)
         return client
 
-    def __login(self, client) -> int:
+    def __login(self, client):
```

Comment:
```suggestion
    def __login(self, client) -> None:
```
How about explicit typing? :)

Reply:
Fixed, thanks